packages/extensions-sdk/src/cli/commands/validators/check-readme.ts

```diff
@@ -0,0 +1,47 @@
+import { opendir } from 'node:fs/promises';
+import { type Ora } from 'ora';
+import type { Report } from '../../types.js';
+
+const checkReadMe = {
+	name: 'readme',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for README';
```

Comment:
A bit more opinionated but should be consistent. If we leave as is L13 should be updated to be inline with the others. 
```suggestion
		spinner.text = 'Check for readme';
```